man/sg.1.xml

```diff
@@ -45,10 +45,15 @@
   <refsynopsisdiv id='synopsis'>
     <cmdsynopsis>
       <command>sg</command>
-      <arg choice='opt'>- </arg>
-      <arg choice='opt'>group
-	<arg choice='opt'>-c </arg>
-	command
+      <arg choice='opt'>
+        <replaceable>-</replaceable>
+      </arg>
+      <arg choice='plain'>
+        <replaceable>group</replaceable>
+      </arg>
+      <arg choice='opt'>
+        <arg choice='opt'><replaceable>-c</replaceable></arg>
+        <replaceable>command</replaceable>
```

Comment:
Ughhh, I don't understand how this XML will render in the screen.  Could you paste the before and after, just to see what's changing?

It gets even worse because my system uses sg(1) from util-linux, so I don't have this manual page installed.

Reply:
Thanks!  Then LGTM.  Although I think it would be good to split the different change of this commit, and document each one well in the commit message.